### DISCUSSION TITLE
What is the order of names? And how can I change it?

### DISCUSSION THREAD

**Discussion by HsunGong** [photoprism/photoprism#3098] [Q&A]
May I ask the order of people's names while assigning the name to face.

One situation is that I assign a new name to a new face, and the new name become the `first` one of the name list, which is annoying. Because this new name is actually one of my friend, but most photos are my family (that I need to assign, also `high frequency names`). And I had to type one or more chars for searching the most used (`high frequency`) names.

So how can I change the order of names? Can I change it from the database? Or manually assign/change the order?

**Comment by graciousgrey** [comment]
It's not possible yet to change this. However we do have a related feature request: 
- https://github.com/photoprism/photoprism/issues/1865
